Tests/UnitTests/Algebra/SolveTest/SolveOneEquation.cs

```diff
@@ -35,7 +33,8 @@ static void AssertRootCount(Set roots, int target)
 
         void TestSolver(Entity expr, int rootCount, Integer? toSub = null, bool testNewton = false)
         {
-            var roots = MathS.Settings.AllowNewton.As(false, () => expr.SolveEquation(x));
+            var roots = MathS.Settings.AllowNewton.As(false /* shouldn't it be testNewton instead of false? */, 
```

Comment:
Newton Solver is tested via SolveNt below

Reply:
The idea is that the number of analytical solutions and numerical solutions should be the same